charts/incubator/librespeed/values.yaml

```diff
@@ -13,6 +13,11 @@ image:
   # -- image pull policy
   pullPolicy: IfNotPresent
 
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
```

Comment:
Not giving any podSecurityContext is enough (defaults to 0, in a more stable way for some reason)

Reply:
Yea, just wanted to test, ill remove it after and  on questions.yaml also